### PR TITLE
refresh situation on Qt Windows packages

### PR DESCRIPTION
**Pull request by stonebig**

### CODE REVIEW

**Comment by ccordoba12** [comment]
Sorry, I don't agree with this. We don't want to start recommending alpha stuff to run qtconsole. Please reopen when PySide2 is officially released.

File: README.md [hunk 1]
```diff
@@ -38,16 +38,21 @@ To install:
...
 ### Installing Qt (if needed)
 We recommend installing PyQt with [conda](http://conda.pydata.org/docs):
 
     conda install pyqt
 
 or with a system package manager. For Windows, PyQt binary packages may be
-used.
+used:
+
+    pip install pyqt5
```
Comment (takluyver) [inline README.md]: Actually, this now works (or should work) on 64-bit Linux and Mac as well.

File: README.md [hunk 2]
```diff
@@ -38,16 +38,21 @@ To install:
...
+
+or
+
+    pip install http://download.qt.io/snapshots/ci/pyside/5.11/latest/pyside2/PySide2-5.11.0a1-5.11.0-cp36-cp36m-win_amd64.whl
```
Comment (takluyver) [inline README.md]: Does this work with `https`? We should never recommend installing software from an unsecure connection unless there's no other option (and if there's no other option, we should be trying to figure one out ASAP).

**Comment by takluyver** [comment]
Oh, and I missed that it was an alpha. I'd agree with Carlos that it's not great to point people to alpha software.

**Comment by ccordoba12** [comment]
PySide2 is going to be officially released with Qt 5.11, scheduled by the end of this month. I hope by then the Qt guys will upload PySide2 to PyPI, so there's no need to point people to an alpha wheel in an insecure location.

File: README.md (hunk 1 above)
Comment (stonebig) [reply README.md]: I don't have a Mac nor a Linux VM, so I didn't dare writing about that. Maybe you can complete the update ?

**Comment by stonebig** [comment]
https changed proof: 
https_pyside2

**Comment by stonebig** [comment]
As I don't know what you can mean per secure, I removed nevertheless the https link.

File: README.md (hunk 1 above)
Comment (takluyver) [reply README.md]: I think it does work, so I'd just remove the phrasing that says it's only for Windows.

**Comment by takluyver** [comment]
Thanks - I think installing from a trusted site with https is OK, but we should leave it until PySide2 is actually released.

**Comment by stonebig** [comment]
as a remark, shouldn't all the other link be changed to https ? even if documentation ?

**Comment by takluyver** [comment]
It's certainly worth doing anywhere that that the https URL works. It's most critical for automated installation (IMO), because human readable information has to pass a sense check before it breaks things. But if you want to make PRs changing links to https, go for it!

**Comment by takluyver** [comment]
Thanks! I assume you've checked that all those URLs work with https?

**Comment by stonebig** [comment]
they all works, but the https://pyside.github.io/docs/pyside was reverting to http. until I put a "/" at the end to remain https

**Comment by takluyver** [comment]
That sounds like a bug in Github's redirect rules, but thanks for checking :-)

File: README.md (hunk 1 above)
Comment (ccordoba12) [reply README.md]: Thomas is right, this works for macOS and Linux as well.

**Comment by ccordoba12** [comment]
Closing because this is really outdated.

**Pull request closed without merging**